lnrpc/rpc.proto

```diff
@@ -1743,10 +1749,16 @@ message ListInvoiceResponse {
     repeated Invoice invoices = 1 [json_name = "invoices"];
 
     /**
-    The index of the last time in the set of returned invoices. Can be used to
-    seek further, pagination style.
+    The index of the last item in the set of returned invoices. This can be used
+    to seek further, pagination style.
     */
     uint32 last_index_offset = 2 [json_name = "last_index_offset"];
+
+    /**
+    The index of the last item in the set of returned invoices. This can be used
+    to seek backwards, pagination style.
+    */
+    uint32 first_index_offset = 3 [json_name = "first_index_offset"];
```

Comment:
Should we modify these to be `uint64` as that's what's used within the invoice add index?

Reply:
According to the "Updating A Message Type" section in https://developers.google.com/protocol-buffers/docs/proto, it should be safe to do this. It's one of 15 things you "just have to remember" lol.

Reply:
Fixed.